src/blockchains/eth/eth_worker.ts

```diff
@@ -31,19 +33,18 @@ export class ETHWorker extends BaseWorker {
 
     this.feesDecoder = new FeesDecoder(this.web3Wrapper);
     this.withdrawalsDecoder = new WithdrawalsDecoder(this.web3Wrapper);
+    this.modes = [];
   }
 
-  async fetchData(fromBlock: number, toBlock: number): Promise<[Trace[], Map<number, ETHBlock>, ETHReceiptsMap]> {
-    return await Promise.all([
-      fetchEthInternalTrx(this.ethClient, this.web3Wrapper, fromBlock, toBlock),
-      fetchBlocks(this.ethClient, this.web3Wrapper, fromBlock, toBlock, true),
-      fetchReceipts(this.ethClient, this.web3Wrapper,
-        this.settings.RECEIPTS_API_METHOD, fromBlock, toBlock),
-    ]);
+  async fetchData(fromBlock: number, toBlock: number): Promise<[Trace[], Map<number, ETHBlock>, ETHReceipt[]]> {
```

Comment:
We fetch raw data depending on what outputs are needed. Some basic raw data like block times is needed for any output.